pos_session_pay_invoice/models/pos_order.py

```diff
@@ -0,0 +1,14 @@
+# Copyright <YEAR(S)> <AUTHOR(S)>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+    
+    @api.model
+    def search_paid_order_ids(self, config_id, domain, limit, offset):
+        domain = [("pos_reference", "!=", False), ("lines", "!=", False)]
```

Comment:
You should add domain to the current domain. Or did I missed something ?

Reply:
hum right, we forgot to add instaed of replacing.

Reply:
Hi @legalsylvain .
Thank you for your feedback.
It's kinda new to me